.flake8

```diff
@@ -19,5 +19,161 @@ extend-exclude =
     build,
     modules,
     .git
+    libraries/AP_Camera/examples/gst_rtsp_server.py
+    libraries/AP_Camera/examples/gst_rtsp_to_wx.py
+    libraries/AP_Camera/examples/gst_udp_to_rtsp.py
+    libraries/AP_Camera/examples/gst_udp_to_wx.py
+    libraries/AP_Declination/generate/generate.py
+    libraries/AP_Declination/generate/testvectors.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/addfunc_parse.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/af_parse_cubemx.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/af_parse.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/bdshot_encoder.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/CKS32F407xx.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/convert_betaflight_unified.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/dma_parse.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/dma_resolver.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F100xB.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F103xB.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F105xC.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F303xC.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F405xx.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F407xx.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F412Rx.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F413xx.py
+    libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F427xx.py
```

Comment:
These are generated files; it might be possible to fix the generator to produce linted python.

Reply:
I will try to do that in a separate PR.